benchmark/buffers/buffer-read.js

```diff
@@ -30,14 +30,14 @@ function main(conf) {
   var len = +conf.millions * 1e6;
   var clazz = conf.buf === 'fast' ? Buffer : require('buffer').SlowBuffer;
   var buff = new clazz(8);
-  var fn = 'read' + conf.type;
+  var fn = `read${conf.type}`;
```

Comment:
I think a lot of benchmarks don't run on v4.x or older anyways...I've seen people on IRC 
asking why some benchmark doesn't run and turns out they are using v4.x to run it